### DISCUSSION TITLE
How to generate an Coolify API Token with more permissions than read data

### DISCUSSION THREAD

**Discussion by Sebasmzt** [coollabsio/coolify#3661] [Q&A] [answered]
Hello, good morning.

I'm looking for a way to generate an api token with  * permissions like the docs says [Authorization Docs](https://coolify.io/docs/api-reference/authorization) . Does anyone know?

I only see `Read Only` and `View sensitive Data` and I can't use `/api/deploy/{uuid}` endpoint with regular API TOKENS. When i'm trying to do it I receive the next response

```js
{
    "message": "You are not allowed to perform this action."
}
```

Thank you

**Comment by andrasbacsai** [comment, marked as answer]
If you are on v349, you need to unselect `Read Only` and `View sensitive Data` to get the `*` permission.
In v350 (releasing soon), the UI is updated:
![image](https://github.com/user-attachments/assets/c4ef8535-2427-46c9-96eb-ba5adb7ecf6f)
